js/libs.js

```diff
@@ -445,6 +470,20 @@ Fliplet.Registry.set('notification-inbox:1.0:core', function(element, data) {
     }
   }
 
+  function showAllNotification() {
+    $('.notification-read, .notification-unread').show();
+  }
+
+  function showUnreadNotification() {
+    $('.notification-read').hide();
+    $('.notification-unread').show();
+  }
+
+  function changeToggleStatus(toggleText, toggle) {
```

Comment:
I don't agree with this approach. Instead of using the function to set the toggle text, the toggle text should be all in the markup. This makes it easier to change the copy or apply localisation.

The function for changing toggle status can be used to change the status, e.g. `all` or `unread` as a key or for changing CSS classes.

Right now with this approach, this function is allowing all kinds of possibilities that shouldn't be allowed, e.g. `changeToggleStatus('Bring me an apple', false)`.

Reply:
Changes are added.